.golangci.yml

```diff
@@ -18,6 +18,17 @@ linters:
     - copyloopvar
     - goimports
     - depguard
+    - asasalint
+    - asciicheck
+    - bidichk
+    - durationcheck
+    - gocheckcompilerdirectives
+    - gochecksumtype
+    - makezero
+    - protogetter
+    - reassign
+    - zerologlint
```

Comment:
Seems we don't use `zerolog`, if we don't use it, I think we don't have to add this linter. Less linters have less check time.

Reply:
nice catch